torch/fx/experimental/proxy_tensor.py

```diff
@@ -654,6 +655,11 @@ def wrapped(*proxies):
             lambda t: get_proxy_slot(t, tracer, t, lambda x: x.proxy),
             out
         )
+        out = pytree.tree_map_only(
+            (torch.ScriptObject, AbstractScriptObject),
+            lambda t: get_proxy_slot(t, tracer, t, lambda x: x),
+            out
+        )
```

Comment:
I'm confused: there should only be AbstractScriptObject here in the `out`. If there are any ScriptObject, then the Fakeification failed to happen

Reply:
Oh, we might not be running make_fx(mode="fake')